server/public/model/client4.go

```diff
@@ -588,6 +588,10 @@ func (c *Client4) bookmarkRoute(channelId, bookmarkId string) string {
 	return fmt.Sprintf(c.bookmarksRoute(channelId)+"/%v", bookmarkId)
 }
 
+func (c *Client4) perfMetricsRoute() string {
+	return "/perf"
+}
```

Comment:
Not a big deal but what's wrong with `client_metrics` or something similar? Perf sounds very generic as it could be used for pretty much anything.

Reply:
We're trying to avoid hitting anything that might be picked up after an adblocker. I was testing with `/metrics`, and that was getting blocked by uBlock

Reply:
Interesting. I am surprised they'd be blocking any requests to the current origin.

Reply:
I think it's just a general rule that adblockers block some patterns right away. But yeah I would expect to let requests to current origin.

Reply:
If the word `metrics` is the problem, would `client_perf` work? Just coming from the same rationale as above, with "perf" being so generic but we have a quite specific use case in mind with this endpoint.

Reply:
I'm totally open to it and I believe that makes sense, any concerns @hmhealey on `client_perf`?